cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/TransactionManagerDebuggerMain.java

```diff
@@ -229,9 +229,18 @@ private void printUsage(boolean error) {
     } else {
       pw = new PrintWriter(System.out);
     }
+    
+    String toolName;
+    String os = System.getProperty("os.name").toLowerCase();
+    if (os.indexOf("win") >= 0) {
+      toolName = TOOL_NAME_BASE + ".bat";
+    } else {
+      toolName = TOOL_NAME_BASE + ".sh";
+    }
+    
     pw.println("Usage:" +
-        "\n\t " + TOOL_NAME + " view [ <option> ... ]" +
```

Comment:
how did it work previously, without the script suffix? Just left to the user to append the suffix as they desire?